en/docs/install-and-setup/configuring-api-manager-for-ob.md

```diff
@@ -131,7 +131,130 @@ account retrieval. By default, this is disabled and the configuration is set to
+
+    ??? tip "Click here to see JWS Response Signing configurations"
+
+         1. Configure the `UKJwsResponseHandlingExecutor` executor. The priority of `UKJwsResponseHandlingExecutor` 
+            must be lower than `ConsentEnforcementExecutor`. For example:
+           ``` toml 
+           [[open_banking.gateway.openbanking_gateway_executors.type.executors]]
+           name = "com.wso2.openbanking.accelerator.gateway.executor.impl.consent.ConsentEnforcementExecutor"
+           priority = 6
+
+           [[open_banking.gateway.openbanking_gateway_executors.type.executors]]
+           name = "com.wso2.openbanking.uk.gateway.executors.jws.UKJwsResponseHandlingExecutor"
+           priority = 999
+           ```
+        
+         2. Enable signing and define the response signing algorithms:
+           ``` toml
+           [open_banking.jws_signature.response_signing]
+           enabled=true
+           allowed_algorithm="PS256"
+           ```
+
+        3. Configure the alias and kid values of the signing certificates:
+            - `signing_cert_alias`: The alias of the signing certificate stored in the keystore. Used to sign responses in a production environment. Default value is `wso2carbon`.
+            - `sandbox_signing_cert_alias`: The alias of the signing certificate stored in the keystore. Used to sign responses in a sandbox environment. Default value is `wso2carbon`.
+            - `signing_cert_kid`: The kid value of the corresponding public key of the private key, which is used for signing in a production environment. Default value is `1234`. Mandatory configuration.
+            - `sandbox_signing_cert_kid`: The kid value of the corresponding public key of the private key, which is used for signing in a sandbox environment. Default value is `5678`. Mandatory configuration.
+           ```toml
+           [open_banking.ob_identity_retriever.server]
+           signing_cert_alias="wso2carbon"
+           sandbox_signing_cert_alias="wso2carbon"
+           signing_cert_kid="w7NFeMODzCMOZen_WECAlG9N8gg"
+           sandbox_signing_cert_kid="w7NFeMODzCMOZen_WECAlG9N8gg"
+           ```
+
+        4. Configure the JWKS size limit and timeout and values:
+           ```toml
+           [open_banking.ob_identity_retriever.jwks_retriever]
+           size_limit=51200
+           connection_timeout=2000
+           read_timeout=2000
+           ```
+
+        5. The default trust anchor used for the signing is `openbanking.org.uk`. To change this value, 
+           add and configure the following:
+           ``` toml
+           [open_banking.uk.signing_config]
+           obie.trusted_anchors.signing = "openbanking.org.uk"
+           obie.org_id="0015800001HQQrZAAX"
+           ```
+        6. Configure the API contexts that require response signing:
+          ``` toml
+          [[open_banking.uk.signing_config.response_sig_required_apis]]
+          api_context=”/open-banking/v3.1/pisp”
```

Comment:
pisp is added by default so no need to mention again. lets add event api

 api_context=”/open-banking/v3.1/event”

Reply:
Fixed in 0913f39d3e68e6d5c0f106f6490a307c24c16758